Tests/test_SffIO.py

```diff
@@ -506,7 +506,7 @@ def fileiter(handle):
     with open("Roche/E3MFGYR02_alt_index_at_start.sff", "rb") as handle:
         records2 = list(SffIterator(handle))
     for old, new in zip(records, records2):
-        assert str(old.seq) == str(new.seq)
+        assert old.seq == new.seq
```

Comment:
self.assertEqual(old.seq, new.seq)

And again two more times later on

Reply:
But that section is not part of a unittest

Reply:
OK, context never obvious from a code snippet